psi4/share/psi4/fsapt/fsapt.py

```diff
@@ -209,14 +209,20 @@ def partitionFragments(fragkeys,frags,Z,Q,completeness = 0.85):
         inds = sorted(range(len(sums)),key=lambda x:-sums[x])
         sum = sums[inds[0]] + sums[inds[1]]
         if sum <= completeness:
-            raise Exception('Orbital %d is not complete over two fragments.' % (a+1))
+            raise Exception('Orbital %d is not complete over two fragments. ' \
+                            'To avoid this error, please try to avoid cutting ' \
+                            'multiple bonds, aromatic rings, etc., in your ' \
+                            'definitions of fragments.' % (a+1))
```

Comment:
Line continuations make me nervous. How about something like the below from [SO](http://stackoverflow.com/a/10660443)?

```
s = ("this is a very"
      "long string too"
      "for sure ..."
     )
```